includes/block.php

```diff
@@ -121,6 +121,8 @@ function build_query_args( $attributes ) {
 		'posts_per_page' => $posts_per_page,
 		'order'          => $attributes['order'],
 		'orderby'        => $attributes['orderBy'],
+		'post_status'    => 'publish',
```

Comment:
I did notice a bunch of private posts in testing, but it slipped my mind. Good call!